gitlab/.gitlab-ci-main.yml

```diff
@@ -1,13 +1,17 @@
 ---
 .static_template: &static_template
   before_script:
-    - composer install --quiet
+    - wget -O - https://raw.githubusercontent.com/simesy/scaffold-tooling/49bd0cd/gitlab/setup-minimal.sh | bash
```

Comment:
@tobybellwood this is basically to avoid running composer outside of the container. Pinned these to specific versions just while unstable. There are a lot of ways we could do this, just ideally not requiring compsoer install which is slow and also not where we usually run composer in the container.